### DISCUSSION TITLE
Electron Overlay unable to load preload

### DISCUSSION THREAD

**Discussion by SuperHeroDev** [community/community#50617] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Today I followed the below youtube video to make Electron CPUusage overlay.
https://www.youtube.com/watch?v=zWuuk_j1iwM&ab_channel=tylerlaceby
I put the same with the video but I got an error.
The error says it couldn't load preload.js.
Any help is appreciated.

**Comment by iamsuiux** [comment, marked as answer]
Hi, I got the same error recently.
I followed everything with the video and it couldn't load preload.
In this case, you need to set nodeIntegration as false for mainWindow preferences.
You can reference the below repo.
https://github.com/doBestMan/electron-overlay

**Comment by queenofcorgis** [comment]
Glad you found a solution ✅  - Wanted to share a reminder the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you.
